xlsx.cabal

```diff
@@ -105,6 +105,7 @@ Library
                    , network-uri
                    , old-locale   >= 1.0.0.5
                    , safe         >= 0.3
+                   , scientific
```

Comment:
What about adding some reasonable bounds, e.g. >= 0.3.6 (the version where Hashable was fixed)?

Reply:
Since we'e only relying on

- `Fractional` instance of `Scientific`,
- the `toRealFloat` function

actually a quite low version bound could suffice. 0.3.0.0 appears to be the birth version of `toRealFloat`.

Reply:
I'd prefer the safer bound